Products/ZenCollector/daemon.py

```diff
@@ -468,6 +482,9 @@ def writeMetric(self, contextKey, metric, value, metricType, contextId,
 
         # check for threshold breaches and send events when needed
         if value is not None:
+            if extraTags:
+                tags.update(extraTags)
```

Comment:
Do we want the extraTags to possibly overwrite the default tags?

Reply:
Yes. That was intentional.